### ISSUE TITLE
error: the server doesn't have a resource type "cluster"

### ISSUE THREAD

**Issue by clifford64** [Heavybullets8/heavy_script#163]
I am using the example crontask on the documentation page and I receive this error after each run. 

error: the server doesn't have a resource type "cluster"

It only started happening after updating to dragonfish and updating heavyscript. As far as I can tell, the script runs normally, it just produces this error after running.

**Comment by heavybullets8** [comment]
It does not cause any real issues. Just makes the output look ugly. This issue comes from this line of code:
https://github.com/Heavybullets8/heavy_script/blob/0729d3d2d59ae58682d84529bd491cecd713d47b/functions/backup_restore/database/cnpg_dump.sh#L123

`--ignore-not-found` no longer appears to be working for Dragonfish, either that, or it never worked as I intended. 

The check is specifically to see if there are any apps on your system using CNPG. Since you do not have any applications like that it results in that error from Kubernetes. Not an issues though. 

Planned to be fixed once the Dragonfish Backup/Restore has been implemented.
